hackathon/activities/containerlab-b-start/README.md

```diff
@@ -102,7 +102,9 @@ sudo docker image list
 ```
 ```
 REPOSITORY                                 TAG         IMAGE ID       CREATED              SIZE
-vrnetlab/vr-sros                           24.3.R3     36a2e22c7267   About a minute ago   766MB
+...
+vrnetlab/nokia_vsr                         24.3.R3     1b8042dbbb41   11 seconds ago       767MB
```

Comment:
mm, let me check this, I think I need to set the image type to `sros` so that the image name would be `nokia_sros` and not `nokia_vsr`

Reply:
fixed vrnetlab to use nokia_sros image name